### ISSUE TITLE
AVT: Operable – Keyboard Accessible |  Icon's are not accessible via keyboard

### ISSUE THREAD

**Issue by kumarivijeta** [carbon-design-system/carbon-components-angular#2831]
## Detailed description

> Describe in detail the issue you're having.

I am showing tooltip on hover of information icon, But when I am trying to access the information icon via keyboard it is not getting focused.

> Is this a feature request (new component, new icon), a bug, or a general issue?

general issue

> Is this issue related to a specific component?
Icon

> What did you expect to happen? What happened instead? What would you like to see changed?

The Icon/elements should be accessible using keyboard as they are for mouse.

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

Carbon 11, carbon-components-angular@5.14.1

## Steps to reproduce the issue

1.Use tab key to go to the information icon

## Additional information

* Screenshots 
<img width="941" alt="Screenshot 2024-03-28 at 1 53 36 PM" src="https://github.com/carbon-design-system/carbon-components-angular/assets/60390189/4d3a578f-b07d-403f-8197-873a327791c8">
or code

**Comment by kumarivijeta** [comment]
using tabindex="0" I am able to do keyboard navigation for icons
